src/Raven.Client/Documents/Session/IAsyncSessionDocumentCounters.cs

```diff
@@ -11,24 +11,28 @@
 namespace Raven.Client.Documents.Session
 {
     /// <summary>
-    ///     Advanced async counters session operations
+    /// Provides async client API for counter operations on a specific entity
```

Comment:
I think it will be good also to add a general info what are the counters and their basic characteristics, with a link to the docs

Reply:
+1